frontend/src/libs/components/app/app.tsx

```diff
@@ -27,9 +27,15 @@ const App: React.FC = () => {
     }
   }, [isRoot, dispatch]);
 
+  const handleSighIn = useCallback(() => {
```

Comment:
handleSignIn function should be inside Header component

Reply:
Great, 100% agree. It was outside in our thread.js task though.